descartes_moveit/src/ikfast_moveit_state_adapter.cpp

```diff
@@ -126,7 +125,7 @@ bool descartes_moveit::IkFastMoveitStateAdapter::getFK(const std::vector<double>
 
 void descartes_moveit::IkFastMoveitStateAdapter::setState(const moveit::core::RobotState& state)
 {
-  assert(static_cast<bool>(robot_state_));
+  ROS_ASSERT_MSG(static_cast<bool>(robot_state_), "'robot_state_' member pointer is null. Have you called initialize()?");
   *robot_state_ = state;
```

Comment:
I may be contradicting myself here but rather that checking the nullity of the ```robot_state_``` pointer why not just call its __reset__ method and use the [RobotState copy constructor](http://docs.ros.org/jade/api/moveit_core/html/classmoveit_1_1core_1_1RobotState.html#ae226721d6aff0342786c2acdc53a5d20) ?